.github/workflows/publish-docker.yml

```diff
@@ -37,7 +34,6 @@ jobs:
       - name: Build and push Docker image
         uses: docker/build-push-action@a66e35b9cbcf4ad0ea91ffcaf7bbad63ad9e0229
```

Comment:
I believe `docker/login-action@v1` should be enough here...

Reply:
No build-push-action? N.B. in edc974e7 I have now added configuration to docker/metadata-action.